### ISSUE TITLE
appl download returns "403 forbidden"

### ISSUE THREAD

**Issue by MaximeBouton** [JuliaPOMDP/SARSOP.jl#12]
The build fails when downloading APPL. I tried to access it manually through a browser and I had an error as well. This is a reproduction of the error message returned during the build:

```julia
julia> download("https://bigbird.comp.nus.edu.sg/pmwiki/farm/appl/uploads/Main/appl-0.96.tar.gz", "appl-0.96.tar.gz")
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 403 Forbidden
ERROR: failed process: Process(`curl -L -f -o appl-0.96.tar.gz https://bigbird.comp.nus.edu.sg/pmwiki/farm/appl/uploads/Main/appl-0.96.tar.gz`, ProcessExited(22)) [22]
Stacktrace:
 [1] pipeline_error(::Base.Process) at ./process.jl:682
 [2] run(::Cmd) at ./process.jl:651
 [3] download(::String, ::String) at ./interactiveutil.jl:600
```

**Comment by mykelk** [comment]
There seems to be a mirror on GitHub [here](https://github.com/personalrobotics/appl).

**Comment by mykelk** [comment]
Here's a direct link: https://github.com/personalrobotics/appl/archive/0.96.tar.gz

**Comment by MaximeBouton** [comment]
I also found this one which seems more recent: [https://github.com/AdaCompNUS/sarsop](https://github.com/AdaCompNUS/sarsop)

**Comment by MaximeBouton** [comment]
We should probably stuck with the archive 0.96 for now. The build is passing now, I did some test on my windows machine and it should work as well. I will close the issue if there are no objections. Thanks @mykelk for the link.

**Comment by mykelk** [comment]
Cool. You can close this with the commit number where you updated the link.

**Comment by MaximeBouton** [comment]
Fix by this commit: [3531bd2963b80451e435a6dd0d5620f42d056b72](https://github.com/JuliaPOMDP/SARSOP.jl/commit/3531bd2963b80451e435a6dd0d5620f42d056b72)
